gcloud_utils/dataproc.py

```diff
@@ -17,9 +17,9 @@ class Dataproc(object):
     """Module to handle with Dataproc cluster"""
 
     def __init__(self, project, region, http=None):
-        self.__project = project
+        self.__project = project_id
         self.__region = region
-        self.__logger = logging.getLogger(name=self.__class__.__name__)
+        self.__logger = logging.getLogger(cluster_name=self.__class__.__name__)
         self.__client = discovery.build('dataproc', 'v1', http=http)
```

Comment:
Hi, please verify your changes.

You removed `discovery` import but keep using it in this line.